core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -54,6 +54,11 @@ public class LocalCacheFileInStream extends FileInStream {
   private final URIStatus mStatus;
   private final FileInStreamOpener mExternalFileInStreamOpener;
 
+  private byte[] mBuffer = null;
+  private final int mBufferSize;
```

Comment:
nits put the final variable together with other final variables.

Reply:
done